### ISSUE TITLE
Configuration for OpenRefine on Windows should use only 1 config file (.ini)

### ISSUE THREAD

**Issue by thadguidry** [OpenRefine/OpenRefine#3057]
**Is your feature request related to a problem or area of OpenRefine? Please describe.**
On Windows, OpenRefine has 2 .ini files that can be used for applying JVM settings.  We should only have 1 .ini file.

**Describe the solution you'd like**
Allow the `openrefine.exe` launching to read settings from the refine.ini file only.
Remove the `openrefine.l4j.ini` file from packaging.

**Describe alternatives you've considered**

**Additional context**

**Comment by wetneb** [comment]
I had a quick look to estimate the difficulty of this task and put a reasonable price tag on it.

The `openrefine.l4j.ini` and `refine.ini` do not use the same syntaxes, so one cannot directly point either `refine.bat` to `openrefine.l4j.ini` or `openrefine.exe` to `refine.ini`.

The name `openrefine.l4j.ini` is hardcoded by Launch4j into `openrefine.exe` so this filename does not seem configurable, see:
https://launch4j.sourceforge.net/docs.html#Additional_jvm_options

Therefore the simplest solution for this would probably be to adapt `refine.bat` so that it uses `openrefine.l4j.ini`, changing it so that it accepts the syntax currently used there. This would be slightly odd in the sense that `refine.bat` would rely on a file with `l4j` in its name while Lanch4j is not used at all by `refine.bat`, but I would say this would still be an improvement over the current situation and still solve this issue.

It should not be too hard to submit a patch to Launch4j to make this filename configurable, but it will probably take a while before this is merged and released, so that could be a follow-up issue on this.

**Comment by wetneb** [comment]
Just for fun I tried to make Launch4j configurable: https://sourceforge.net/p/launch4j/git/merge-requests/20/.
But I still think the easiest solution would be to adapt `refine.bat`.

**Comment by wetneb** [comment]
Good news, my PR in Launch4j was merged so this filename should be configurable in the next Launch4j release. After that it should be possible for Windows and Linux to use the same configuration file. I am not sure about Mac OS though - for now the options seem to be loaded from some `info.plist` according to the docs, and it looks not so easy to change that.

**Comment by wetneb** [comment]
Update: this is still blocked by necessary changes in Launch4j. The following PR needs merging and releasing: https://sourceforge.net/p/launch4j/git/merge-requests/21/
